src/Sylius/Bundle/ResourceBundle/DependencyInjection/Driver/Doctrine/DoctrinePHPCRDriver.php

```diff
@@ -28,6 +32,57 @@ class DoctrinePHPCRDriver extends AbstractDoctrineDriver
     /**
      * {@inheritdoc}
      */
+    public function load(ContainerBuilder $container, MetadataInterface $metadata)
+    {
+        parent::load($container, $metadata);
+        $this->addResourceListeners($container, $metadata);
```

Comment:
Overriding the `load` method here, alternative would be adding another special case or delegating method in the AbstractDriver, but not sure I like that approach more.